language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/helpers/LedgerIdentityServiceImpl.scala

```diff
@@ -13,8 +13,10 @@ import com.daml.ledger.api.v1.ledger_identity_service.{
 }
 import io.grpc.ServerServiceDefinition
 
+import scala.annotation.nowarn
 import scala.concurrent.{ExecutionContext, Future}
 
+@nowarn("msg=deprecated")
```

Comment:
Please move `@nowarn` annotation as close as possible to the call site, ideally at the expression level. And for deprecation don't use the message but `cat=deprecation&origin=<regex-matching-the-deprecation-origin>`.

Reply:
Ok, done